docs/research/wallet-comparison.md

```diff
@@ -0,0 +1,171 @@
+
+- https://medium.com/@ACINQ/introducing-phoenix-5c5cc76c7f9e
+- https://phoenix.acinq.co/faq
+
+### Blue Wallet
+
+In a nutshell: Semi- or Non-custodial (when using own LND) setup and a good UX that includes multiple wallet, but overall not as good as Phoenix.
+
+Setup:
+
+- Plug in own LND node, or use [LndHub](https://github.com/BlueWallet/LndHub)
+  - LndHub is a semi-custodial setup ("minimum trust for endusers")
+- Where is trust involved?
+  - LndHub
+
+Pro:
+
+- Multiple wallet support
+- Vaults (Not tried out, feels like a different feature unrelated to Lightning Wallet)
+
+Pro / Con:
+
+- One-click wallet create for setup, but onboaring not as clear and rich as Phoenix
+
+Con:
+
+- Cannot "empty the wallet", always have to keep at least some sats in for "paying fees"
+  - This is super annoying, it is literally impossible to empty this wallet.
+- Multi-wallet management
+  - The wallets are completely independent; sending money from one to the other is quite cumbersome.
+  - I would have expected that I can easily move funds from one wallet to another, but this is quite cumbersome.
+  - Not sure what the use cases behind multiple wallets is; it does not feel like you average "bank account" app where you divide your money into different accounts for different purpose.
+- Send / Receive are not as intuitive as for Phoenix
+  - When using Phonex it becomes clear how Lightning works, even when one is new to send/receive and creating/paying invoices.
+  - The Blue wallet interface can be confusing, because it is not clear what one has to do to send/receive.
+- When using LndHub the backup is an lndhub link or a QR code. There is not 12-word seed available that can just be copied.
+  - How would I scan a QR code for backup with only one phone?
+  - Unclear what the "backup link" represents.
+
+For details check these links:
+
+- https://bluewallet.io/features/
+
+### Breez
+
+In a nutshell: Non-custodial setup similar to Phoenix and a good UX; includes features that go beyond just a wallet.
+
+I think Breez is currently the biggest competitor to what we have planned, because they already include "trading services".
+
+Setup:
+
+- Non-custodial setup similar to Phoenix but with more control options
+  - Run own LND node on Phone (forked version of LND + Neutrion Bitcoin node)
+    - Looks like they are the only one running both LND and Neutrino on the phone
+  - Channel management through Async nodes
+  - Can configure own Electrum Server
+  - Cannot configure own Lightning node (outside the phone setup)
+- Where is trust involved?
+  - Channel management
+  - Note: Backup is independent of Breez (different than Phoenix)
+
+Pro:
+
+- Fund wallet with BTC
+- Fund wallet with Fiat: Buy BTC feature enabled through MoonPay integration
+- Channel management
+  - Default
+  - Possibility to interact with Lightning node through basic interface for devs in Preferences
+
+Pro / Con:
+
+- "Marketplace" wallet integrations
```

Comment:
IMHO that's definitely a pro - you can do more things with the wallet.

Reply:
I agree, Somehow :D 

The integrations are a great idea, but sometimes don't work very well - how would you complain about that with Breez? This is cool on paper, but when using it in reality it is not the best UX in the world. I also had trouble understanding what some of the services are - there is no explanation, but the app just opens when clicking on it and connects to your wallet. Not sure I want that - not sure others want that. 

Again, I don't want to talk this down, I think these are powerful and cool features, but I also think we can learn from what be made better.

Reply:
for me this is the "wrong" section - IMHO the ability to have 3rd party integrations using an open standard is a pro (not in "pro/con" section).

as for the explanation, this hinges on the fact that the user knows what the want to use, e.g. by researching earlier the apps they would like to try:
https://www.lapps.co - here you have explanations of all these apps with comments and upvotes.

All I'm saying is that it's an established standard and by deliberately not taking this into account we might be alienating a huge potential userbase for no reason (instead of actually letting the user to decide whether they would like to engage with that or not).

Reply:
case it point: there's 9 Android wallets according to https://www.lapps.co/wallets https://www.lapps.co/wallets

I see some merit in deliberately curating the apps for a polished experience, but this sounds a lot like Apple approach - and not sure whether the majority of users would actually want that. Perhaps some user interviews would be helpful.

Reply:
Fair enough, thanks for the link to the standard, I'll have another look at that and mention it in the Pros!